Function/BookFunc/Helpers/Constants.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TeamsMeetingBookFunc.Helpers
+{
+    internal class ConfigConstants
+    {
```

Comment:
Add a comment to remember that if we change the values of those constants, we MUST update the ARM template as well